### ISSUE TITLE
Add an example for the windowRatio() method

### ISSUE THREAD

**Issue by SableRaf** [processing/processing-examples#14]
The windowRatio() function and its associated variables `rwidth`, `rheight`, `rmouseX`, `rmouseY` are demonstrated in the [release notes for Processing 4.0 beta 7](https://github.com/processing/processing4/blob/58ff6a3ca5c9c2f26a81d4014edc5d5ba0abf4c7/build/shared/revisions.md#processing-40-beta-7
) with the code below but we don't yet have an example built-in.

```Java
void setup() {
  windowResizable(true);
  windowRatio(1280, 720);

  cursor(CROSS);
  strokeWeight(10);
}

void draw() {
  background(240);
  fill(255);
  rect(0, 0, rwidth, rheight);

  fill(0);
  textAlign(CENTER, CENTER);
  textSize(200);
  text(rmouseX + ", " + rmouseY, rwidth/2, rheight/2);
}```

**Comment by benfry** [comment]
Yeah, and that one is more of a “testing” thing that I was using to demonstrate what was going on. 

Also need to note in the (eventual) documentation that it works for *most* things or at least *lots* of things, but there are definitely exceptions.

For instance, the sketch that Dan was using on Coding Train this week was like a perfect example of the worst kind of code to adapt this way. :) Not only is it 3D, which can be funky to get right, it also sets up its own camera/project using `ortho()`, which would need special care as well.
